src/DiabloInterface/Gui/Controls/HorizontalLayout.cs

```diff
@@ -122,10 +122,17 @@ protected override void UpdateRuneList(ApplicationSettings settings, IReadOnlyLi
         {
             if (activeRuneLayoutPanel == null) return;
 
-            activeRuneLayoutPanel.Visible = runes.Count > 0;
             activeRuneLayoutPanel.Controls.Clear();
-            runes.ForEach(rune => activeRuneLayoutPanel.Controls.Add(
-                new RuneDisplayElement(rune, settings.DisplayRunesHighContrast, false, false)));
+            if (runes == null)
```

Comment:
i had the following settings:

character default, difficulty normal
character default, difficulty nightmare

then i changed in main window menu to hell and got the exception